File: Laboratorio10/Laboratorio104/Program.cs
using System;

namespace Laboratorio104
{
	class Program
	{
		static void Main(string[] args)
		{
			Perro miPerro = new Perro("Firulais", 3);
			Console.WriteLine($"Nombre: {miPerro.Nombre}, Edad: {miPerro.Edad}");
			miPerro.Ladrar();
		}
	}
}

File: Laboratorio10/Laboratorio104/Program.vb
Module Program
    Sub Main()
        Dim miPerro As New Perro("Firulais", 3)
        Console.WriteLine($"Nombre: {miPerro.Nombre}, Edad: {miPerro.Edad}")
        miPerro.Ladrar()
        Console.ReadKey()
    End Sub
End Module
